### PR TITLE
feat(content): homepage updates

### PR DESCRIPTION
**Pull request by kennylam**
### Related Ticket(s)

#1330 

### Description

Update homepage with new content and dark theme.

### Changelog

**New**

- homepage content updates
- various image and icon assets

**Changed**

- code cleanup

**Removed**

- duplicate icon assets

### CODE REVIEW

**Comment by RichKummer** [comment]
Looking really good! I noticed just a few things as I was going through:
* The link for Carbon Design System currently takes you to our Resources page - should take you to https://carbondesignsystem.com/
* There should be a down-arrow icon below the header text per the design. Is this possible in Gatsby?

**Comment by oliviaflory** [comment]
@kennylam Caught one more thing in addition to Rich's comments: 
- The card link for IBM Design Language is linking the the IBM Web Standards.

Also noticed 43 files changed, is that from the code cleanup ??

**Comment by kennylam** [comment]
@RichKummer @oliviaflory Thanks for the comments! Everything should be addressed.

Olivia, the large number of file changes are from the cleanup. The majority were duplicate images in multiple folders that were removed and consolidated.

File: src/gatsby-theme-carbon/components/Footer/index.js [hunk 1]
```diff
@@ -14,16 +14,17 @@ const CustomFooter = () => {
   const Content = () => (
     <>
       <p>
-        Have questions? Open an issue in{' '}
+        Have questions? <a href="mailto:digdes@us.ibm.com/">Email us</a> for
```
[the remark this answers is not in the document]
Comment (kennylam) [reply src/gatsby-theme-carbon/components/Footer/index.js:17]: @oliviaflory that's a good point. I'm not sure anyone actively monitors that address. It probably makes more sense to open a `questions` template as we have in the Carbon for IBM.com repo and link users there instead. I'll take care of that in a separate PR.

**Pull request merged**